css/style.css

```diff
@@ -16,4 +16,396 @@ body {
+    margin: 0 auto;
+    padding: 0.5em 0;
+    width: 80%;
+    max-width: 400px;
+    align-items: center;
+    border-top: 1px solid #f3e7e7;    
+}
+.footer-container h4{
+    font-size: 1.2em;
+    font-weight: 300;
+    letter-spacing: 0.1em;
+    margin-bottom: 0.5em;
+}
+.social-links{
+    display:flex;
+    flex-direction: row;
+    flex-wrap:wrap;
+    padding:0;
+    margin:0 0.9em;
+    justify-content:center;
+}
+.social-links a{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 2.80em;
+    margin-right: 1em;
+    margin-bottom: 1rem;
+    width: 2.80em;
+    border:1px solid #9c9c9c;
+    border-radius: 25%;
+    padding: 0.9em;
+    color:rgb(238, 255, 0);
+    background-color: rgb(223, 202, 202);
+   transition: all 0.5s ease-in-out;
+}
+.social-links img{
+    height: 2em;
+    margin: 0 0.5em;
+} 
+.copyright{
+    font-size: 0.8em;
+    color: #424242;
+    margin-top: 0.5rem;
+}
+
+/**************** media query styles ***********************/
+
+@media (min-width:641px){
+    #intro{
+        height: 350px;
+    }
+    /*service section*/
+
+    .service-card img{
+    width: 150px;        
+    margin: 10px 15%;
+    }         
+}
+    /********/
+@media (min-width:961px){
+    .logo_pic{
+        height: 8vh;          
+}   
+    .nav-link{
+        display:flex ;
+        flex-direction: row;
+        position: relative;
+        justify-content: flex-end;
+        background-color: var(--primary);
+        top:0;
+        text-align: center;
+        margin-right: 25px;
+        transition: none;
+}
+    .nav-item{
+        padding: 0.6em;
+        margin: 0.5em;
+}
+    .toggle-button{
+        display: none;
+}
+    /*intro section*/
+    
+    #intro{
+        height: 600px;
+}
+
+    .title{
+        font-size: 4em;
+        letter-spacing: 0.1em;
+}
+    .description{
+        font-size: 1.8em;
+}
+    #intro .btn{
+        font-weight: 500;
+        padding:0.5em;
+        font-size: 1.5em;
+}
+    /*service section*/
+
+    #service-container h2{
+        text-align: center;
+        margin-top: 0.7em;
+        font-weight:400 ;
+        font-size:2em ;  
+    }
+    .cards {
+        display: flex;
+        flex-flow: row wrap;
+        justify-content: space-evenly;
+    }
+    .service-card{
+        max-width: 40%;
+        padding: 2em;
+    }
+    .service-card img{
+        max-width: 360px;       
+        margin: 10px 5%;
+    }
+    .service-card h3{
+        font-size: 1.7em;
+        font-weight: 300;
+        margin:0.5em 0;
+    }
+    /*get karma*/
+    
+    #get_karma{
+        background-color: rgb(241, 229, 205);
+        display: grid;
+        margin-bottom: 2em;
+    }
+    #get_karma img{
+        grid-column: 1 / 2;
+        grid-row: 1 / 2;
+    }
+    #get_karma h4{
+        grid-column: 2 / 4;
+        grid-row: 1 / 2;
+        justify-self: center;
+        align-self: center;
+        display: flex;
+        flex-direction: column;
```

Comment:
This header element does not contain any children - it is a part of the "get karma" layout and does not control layout itself, so should not be a flexbox. Remember `display: flex` is used on the flex container - it will then change how the child elements are displayed

Reply:
Hi Paul, thank you for pointing this out. I think now I have slightly better understanding of how to use flex box. I made the change now. However I wanted to look the h4 in a certain way in the column in a bigger screen. Couldn't figure out how to do it . Regards.